src/components/card/Card/Card.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react'
+import classnames from 'classnames'
+
+interface CardProps {
+  header?: React.ReactNode
+  media?: React.ReactNode
+  footer?: React.ReactNode
```

Comment:
It's my impression that `header`, `children`, and `footer` should be required (with only `media` being optional). We might want to test how the Card looks when any of those are omitted to see if there are repercussions in the styling if we want them to be optional.